sdk/src/store.rs

```diff
@@ -2736,7 +2736,7 @@ impl Store {
                     final_assertions.push(User::new(&label, &data).to_assertion()?);
                 }
                 DynamicAssertionContent::Binary(format, data) => {
-                    todo!("Binary dynamic assertions not yet supported");
+                    //final_assertions.push(EmbeddedData::to_binary_assertion(&EmbeddedData::new(&label, format, data))?);
```

Comment:
Is this meant to be commented?

Reply:
yes, I want to wait to enable that later when I have more time to test